src/averageAges.js

```diff
@@ -15,6 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const arrayOfMen = century
+    ? people.filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    : people.filter(person => person.sex === 'm');
+
+  const menAverageAgeWithoutCentury = arrayOfMen.reduce(function(average, man) {
```

Comment:
Consider using more descriptive variable names, like `menAverageAge` instead of `menAverageAgeWithoutCentury`. 🤔